assets/javascripts/discourse/templates/modal/assign-user.hbs

```diff
@@ -60,16 +60,16 @@
 </DModalBody>
 
 <div class="modal-footer">
-  {{d-button
-    label=(if
+  <DButton
+    @label={{(if
       model.reassign
       "discourse_assign.reassign.title"
       "discourse_assign.assign_modal.assign"
-    )
-    icon=inviteIcon
-    action=(action "assign")
+    )}}
+    @icon={{inviteIcon}}
+    @action={{(action "assign")}}
```

Comment:
Same question about `{{}}` and also is it possible to do `this.assign` here instead or it doesnt work in this case?